src/xhr.js

```diff
@@ -387,6 +387,15 @@ class FetchResponse {
   }
 
   /**
+   * Drains the response and returns a promise that resolves with the response
+   * text.
+   * @returns {!Promise.<string>}
+   */
+  text() {
+    return this.drainText_();
+  }
```

Comment:
actually, looking through the full code again, this is probably better. (adding this new `public` text() method)